pixelgl/gltriangles.go

```diff
@@ -75,7 +75,6 @@ func (gt *GLTriangles) SetLen(len int) {
 	if len < gt.Len() {
 		gt.data = gt.data[:len*gt.vs.Stride()]
 	}
-	gt.submitData()
```

Comment:
The performance impact is *large*. Like, before this, the external calls were 15% of runtime, after, about 9%. And in every use case I could find, there will be a submitData() soon after SetLen is called. So the typical use pattern is

x.SetLen()
copy items into newly allocated space
call x.submitData()

And calling submitData() before the actual data has been copied in just sends a bunch of "empty" vertex data over to GL, so far as I can tell. (I'll follow up in chat.)